pet-race-devops/docker/cassandra/Dockerfile

```diff
@@ -12,26 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.4
+FROM ingenieux/alpine-java8-supervisord
 
-ENV DUMB_INIT_VER=1.0.3 CASS_VERSION=3.5 CASS_DIR=/opt/cassandra JAVA_HOME=/usr/lib/jvm/java-1.8-openjdk/jre LANG=C.UTF-8 PATH=$PATH:$JAVA_HOME/bin
+ENV DUMB_INIT_VER=1.0.3 CASS_VERSION=3.5 CASS_DIR=/opt/cassandra
 
 COPY files/ /
 
+RUN echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf
+
 RUN set -ex \
-  && apk add --no-cache openjdk8-jre bash \
+  && apk add --no-cache python python-dev jemalloc jemalloc-dev bash procps musl musl-dev \
   && apk add --no-cache --virtual .fetch-deps curl \
+  && apk --no-cache add ca-certificates \
+  && mv /sgerrand.rsa.pub /etc/apk/keys/sgerrand.rsa.pub \
+  && apk add --allow-untrusted glibc-2.23-r1.apk \
+  && cp /lib/libc.musl-x86_64.so.1 /opt/jdk/lib/amd64/libc.musl-x86_64.so.1 \
   && adduser -S cassandra \
```

Comment:
Gotta run as root currently.  Not able to lock mem as cassandra in docker, on k8s